DigitalLearningSolutions.Data.Tests/Services/DelegateApprovalsServiceTests.cs

```diff
@@ -20,6 +20,7 @@ public class DelegateApprovalsServiceTests
         private IUserDataService userDataService = null!;
         private ICustomPromptsService customPromptsService = null!;
         private IEmailService emailService = null!;
+        private ICentresDataService centresDataService = null!;

```

Comment:
Could this just be `A.Fake` rather than `null!` (+ some others too)

Reply:
Their values are set (most of them to `A.Fake`) in `SetUp()`.

Reply:
But is there a reason we don't set the value here (to make the setup a bit neater)?

Reply:
Not that I know of, but it's done in SetUp in other test files so I just did it that way for consistency.

Reply:
We need new mock objects for each test case; if we initialise them at declaration they'll be reused as the fixture class is only instantiated once.

Reply:
Ah, I see! And I guess clearing the mock at the end is equivalent to creating the mock in the setup! That's fair - happy to leave as is! 🙌